js/main.js

```diff
@@ -152,14 +158,11 @@ function updateFeedback(response, player) {
       <span>${subchunk.toUpperCase()}! ${playerName} is back in the game!</span>
     `
     undoSingleDeal()
-    toggleHighlighting(player)
   } else if (response === 'winner') {
     chunk = `
       <span>${response.toUpperCase()}! ${playerName} wins the game!</span>
     `
 
-    checkLocalStorage();
-
     player1front.classList.remove('hidden');
```

Comment:
You do this same thing on lines 135 - 139, you could make this into a helper function and just call it as needed.

Reply:
I have to call it inside this specific block because there is an automatic loser when a player who has no cards slaps something other than a jack.